phpBB/phpbb/install/controller/installer_index.php

```diff
@@ -0,0 +1,79 @@
+<?php
+/**
+ *
+ * This file is part of the phpBB Forum Software package.
+ *
+ * @copyright (c) phpBB Limited <https://www.phpbb.com>
+ * @license GNU General Public License, version 2 (GPL-2.0)
+ *
+ * For full copyright and license information, please see
+ * the docs/CREDITS.txt file.
+ *
+ */
+
+namespace phpbb\install\controller;
+
+class installer_index
+{
+	/**
+	 * @var helper
+	 */
+	protected $helper;
+
+	/**
+	 * @var \phpbb\language\language
+	 */
+	protected $language;
+
+	/**
+	 * @var \phpbb\template\template
+	 */
+	protected $template;
+
+	/**
+	 * @var string
+	 */
+	protected $phpbb_root_path;
+
+	/**
+	 * Constructor
+	 *
+	 * @param helper 					$helper
+	 * @param \phpbb\language\language	$language
+	 * @param \phpbb\template\template	$template
+	 * @param string					$phpbb_root_path
+	 */
+	public function __construct(helper $helper, \phpbb\language\language $language, \phpbb\template\template $template, $phpbb_root_path)
+	{
+		$this->helper = $helper;
+		$this->language = $language;
+		$this->template = $template;
+		$this->phpbb_root_path = $phpbb_root_path;
+	}
+
+	public function handle($mode)
+	{
+		switch ($mode)
```

Comment:
Actually these are different routes, just in one controller (as the controller logic is the same). So separate controllers or just leave it as it is?